modules/face/data/mean_shape.txt

```diff
@@ -0,0 +1,195 @@
+137.856,219.266
```

Comment:
For different bounding box I have added a function to calculate the relative mean shape. If we calculate mean shape everytime with training data that would be an overhead. SO instead of calculating we would just have to calculate the relative shape according ti the bounding box.